src/gluonts/dataset/common.py

```diff
@@ -455,7 +455,11 @@ def __call__(self, data: DataEntry) -> DataEntry:
 
 
 def load_datasets(
-    metadata: Path, train: Path, test: Optional[Path]
+        metadata: Path,
```

Comment:
The indentation here could be reduced, I think

Reply:
Must have been my IDE's default settings. This should be fixed now.

Reply:
Ha okay, now it should be all good. I ran `black src` as suggested by the pipeline.